### DISCUSSION TITLE
academic email address

### DISCUSSION THREAD

**Discussion by Krydelmany** [community/community#69046] [GitHub Education]
### Hi everyone,

I don't have a academic email and it doesn't
made it available, but github asks, what do I do?

**Comment by mantasu** [comment]
Hi, I would suggest opening a [ticket](https://support.github.com/request/education) as you may have to provide additional proof.

**Comment by Abkeyev99** [comment]
Click "Marked as answer" Thanks so good :)

**Comment by anasjawed283** [comment]
You can use personal mail ID and give a unique identification, did the same and it worked.
